Gems/Atom/RHI/Code/Source/RHI/ShaderResourceGroupPool.cpp

```diff
@@ -257,7 +257,6 @@ namespace AZ
             for (uint32_t i = interval.m_min; i < interval.m_max; ++i)
             {
                 ShaderResourceGroup* group = m_groupsToCompile[i];
-                AZ_PROFILE_SCOPE(RHI, "CompileGroupsForInterval %s", group->GetName().GetCStr());
```

Comment:
Please add this one back as we have an SRG group taking over 2 ms to compile and this was added so it could be tracked.

Reply:
Restored it, the others were way more important.